viewing_party/party.py

```diff
@@ -0,0 +1,208 @@
+# test_wave_01
+def create_movie(movie_title, genre, rating):
+    movie={}
+    if movie_title == None or genre == None or rating ==None :
+        return None
+    else :
+        movie["title"]=movie_title
+        movie["genre"]=genre
+        movie["rating"]=rating
+    return movie
+        
+
+def add_to_watched(user_data, movie):
+ 
+    if len(user_data["watched"]) == 0:
+    
+        user_data["watched"]=[movie]
+    elif len(user_data["watched"]) == 1:
+        return 1
+    return user_data
```

Comment:
This makes a new list in the user data holding only the single movie data, but do we need to make a new list here? `list` has a method that we can use to add new data to an existing list. What if we used that here? Could that handle all cases regardless of the length of the list?

If we remove the `if` branch, could we also remove the special case for size 1 lists? There isn't a test that looks for behavior like this (no test starts with 1 item in the watched list), but maybe this was added after seeing the `assert len(updated_data["watched"]) is 1` in the `add_to_watched` test? That check is interested in the length of the watched list in the returned `user_data`, which _should_ be handled if we consider changing how we work with the list.